UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/YefimYakunin/YeYa_PetsViewController.swift

```diff
@@ -1,8 +1,173 @@
 import UIKit
 
-class YeYa_PetsViewController: UIViewController {
-
+final class YeYa_PetsViewController: UIViewController {
+    @IBOutlet private weak var filtersCollectionView: UICollectionView!
+    @IBOutlet private weak var petsCollectionView: UICollectionView!
+    var pets: [YeYa_PetModel] = []
```

Comment:
make all properties that you can as private

Reply:
@IBOutlet private weak var filtersCollectionView: UICollectionView!
    @IBOutlet private weak var petsCollectionView: UICollectionView!
    private var pets: [YeYa_PetModel] = []
    private var filters: [YeYa_PetFilterModel] = []
    private var filteredPets: [YeYa_PetModel] = []
    private var selectedFilterType: YeYa_PetType = .allPets

All variables are now private